tests/test_library_conf.py

```diff
@@ -2,12 +2,12 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
+import re
 from utils import coverage, weblog, interfaces, released, irrelevant, scenarios
 
-
 # basic / legacy tests, just tests user-agent can be received as a tag
 @irrelevant(library="cpp")
-@released(java="?", php="0.68.2", python="0.53")
+@released(java="?", php="0.68.2", python="0.53", golang="1.53.0")
```

Comment:
it must be declared only once, as you did it in manifest, you can remove this one, same for other `@released`